ports/rp2/boards/RPI_PICO_W/mpconfigboard.h

```diff
@@ -8,6 +8,11 @@
 #define MICROPY_PY_NETWORK 1
 #define MICROPY_PY_NETWORK_HOSTNAME_DEFAULT     "PicoW"
 
+// Enable PPP networking.
+#define MICROPY_PY_NETWORK_PPP_LWIP             (1)
```

Comment:
Could probably put this line in `mpconfigport.h` as:
```c
define MICROPY_PY_NETWORK_PPP_LWIP (MICROPY_PY_NETWORK)
```
Then it's available on all network-enabled boards, eg Pico 2 W.